generateSecrets.js

```
@@ -0,0 +1,33 @@
+const crypto = require('crypto');
+
+function generateSecret() {
+  return crypto.randomBytes(64).toString('base64'); 
+}
+
+const accessSecret = generateSecret();
+const refreshSecret = generateSecret();
+const resetSecret = generateSecret();
+const confirmationSecret = generateSecret();
+
+const envFilePath = '.env.local';
```

ESLint

[error] Extra semicolon.

(semi)